app/javascript/mastodon/features/status/components/card.js

```diff
@@ -194,7 +194,7 @@ export default class Card extends React.PureComponent {
     const provider    = card.get('provider_name').length === 0 ? decodeIDNA(getHostname(card.get('url'))) : card.get('provider_name');
     const horizontal  = (!compact && card.get('width') > card.get('height') && (card.get('width') + 100 >= width)) || card.get('type') !== 'link' || embedded;
     const interactive = card.get('type') !== 'link';
-    const className   = classnames('status-card', { horizontal, compact, interactive });
+    const className   = classnames('status-card', { horizontal, compact, interactive, large: displayLargePreviews });
```

Comment:
should `large` be set to true only if `card.image` is truthy, to handle links without images? For ex. https://mstdn.social/@meg@dair-community.social/109439708179854100